test-interface-sbt-defs/src/main/scala/sbt/testing/Framework.scala

```diff
@@ -34,4 +34,13 @@ trait Framework {
   def runner(args: Array[String],
              remoteArgs: Array[String],
              testClassLoader: ClassLoader): Runner
+
+  /** Scala Native specific: Creates a slave runner for a given run.
+   *  Ported from Scala.JS
+   *  The slave may send a message to the master runner by calling `send`.
+   */
+  def slaveRunner(args: Array[String],
```

Comment:
Important: this method name cannot be changed. It must keep the name `slaveRunner` for compatibility with Scala.js. This is a public to-be-overridden API that testing frameworks must implement. Currently testing frameworks can have a unique implementation of `Framework` for JVM, JS and Native. If we change that method name, they will need a separate source file for Native.

Similarly, Scala.js *cannot* change the name of that method in its API either, since that would be a binary breaking that would break all published testing frameworks, which is not acceptable.

(And no, introducing another method and deprecating this one as a forwarder is not an option, because testing frameworks must *implement* that method, not call it.)

Reply:
I think a comment to that effect is good so any reader can see that our hands are tied and we can't rename it.